orka-ipsw-setup.sh

```diff
@@ -0,0 +1,245 @@
+    
+    # Start SSH service
+    sudo launchctl load -w /System/Library/LaunchDaemons/ssh.plist 2>/dev/null || true
+    
+    log "Remote Login enabled with full disk access for admin user"
+}
+
+# Configure macOS updates (Set to download-only for Tahoe)
+configure_macos_updates() {
+    log "Configuring macOS updates for Download Only..."
+    
+    # Set automatic update check to true, but disable automatic installation
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticCheckEnabled -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticDownload -bool true
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist AutomaticallyInstallMacOSUpdates -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist CriticalUpdateInstall -bool false
+    sudo defaults write /Library/Preferences/com.apple.SoftwareUpdate.plist ConfigDataInstall -bool false
+    
+    # Disable automatic installation of app updates from App Store
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdate -bool false
+    sudo defaults write /Library/Preferences/com.apple.commerce.plist AutoUpdateRestartRequired -bool false
+    
+    log "macOS updates configured for Download Only mode"
+}
+
+# Disable FileVault (When running on Tahoe)
+disable_filevault() {
+    log "Ensuring FileVault is disabled..."
+    
+    # Check if FileVault is enabled
+    local filevault_status=$(sudo fdesetup status | head -1)
+    
+    if [[ "$filevault_status" == "FileVault is On." ]]; then
+        warn "FileVault is currently enabled. For Tahoe, it should be disabled."
+        warn "Please disable FileVault manually through System Preferences > Security & Privacy > FileVault"
+        warn "This requires a reboot and cannot be automated safely."
+    else
+        log "FileVault is already disabled"
+    fi
+}
+
+# Download and install Orka VM Tools
+install_orka_vm_tools() {
+    log "Installing Orka VM Tools version ${ORKA_VM_TOOLS_VERSION}..."
+    
+    local pkg_url="https://orka-tools.s3.amazonaws.com/orka-vm-tools/official/${ORKA_VM_TOOLS_VERSION}/orka-vm-tools.pkg"
+    local pkg_path="/tmp/orka-vm-tools.pkg"
+    local pkg_name="orka-vm-tools.pkg"
+    
+    # Download the Orka VM Tools package
+    log "Downloading Orka VM Tools from $pkg_url..."
+    curl -fsSL "$pkg_url" -o "$pkg_path" || error "Failed to download Orka VM Tools package"
+    
+    # Verify the download was successful
+    if [[ ! -f "$pkg_path" ]]; then
+        error "Orka VM Tools package not found after download"
+    fi
+    
+    # Install the package
+    log "Installing Orka VM Tools package..."
+    sudo installer -pkg "$pkg_path" -target / || error "Failed to install Orka VM Tools package"
+    
+    # Clean up installer
+    rm -f "$pkg_path"
+    
+    log "Orka VM Tools v3.5.0 installed successfully"
```

Comment:
Shouldn't we log the correct version? As it is now a variable. Not hardcode v3.5.0 here